tutorials/01-Introduction.ipynb

```diff
@@ -0,0 +1,2872 @@
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "b6226595-e75a-4759-a541-5b46f8f151d5",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from sympy import *"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2b357c84-685f-4e39-8e22-92c94d7d5e65",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "Let's try out the sine function from `math, numpy` and `sympy`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "id": "37ea3f6c-dcc3-4acc-8a55-0f5d87bd5f5a",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import math\n",
+    "import numpy"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "id": "605102a1-8dd3-4f92-8f8c-3752acc87ab5",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.7071067811865475"
+      ]
+     },
+     "execution_count": 31,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "math.sin(math.pi / 4)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "id": "c4f2e1d7-46f0-4bf1-a5a9-eed8d3b4dd33",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.7071067811865475"
+      ]
+     },
+     "execution_count": 32,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "numpy.sin(numpy.pi / 4)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "id": "0956bd4c-cd7b-445c-b4a0-359b4a4b37eb",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$\\displaystyle \\frac{\\sqrt{2}}{2}$"
+      ],
+      "text/plain": [
+       "sqrt(2)/2"
+      ]
+     },
+     "execution_count": 33,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "sin(pi / 4)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7155302c-a734-42cb-966c-af61b3f39a76",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "We note that you don't always need star import (`*`), and you can disambiguate the modules by:\n",
+    "\n",
+    "- Prefixing the module `sympy.`\n",
+    "- Selectively import `import sin from sympy`\n",
+    "\n",
+    "For sake of making your script longer, but can makes your script less ambiguous and better quality in production."
```

Comment:
People should always use these for production code. `import *` should only be used for interactive sessions. We use it in the tutorial because we consider that to be an interactive session and it keeps down the typing.